no-computer.md

```diff
@@ -1,69 +1,60 @@
+
+The general rule in doubles is that when you lift (hit a high shot), partners move from the front/back position to a side/side position to cover as much area of the court as possible and allow for more reaction time. This is why it is so important to signal to your partner where you intend to serve so that the team can be well-prepared for the rally without losing the element of surprise. 
+
+Without these important hand-signs your partner could be surprised and you may easily lose the rally. 
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
-
+As I mentioned before it prepares your partner for the next shot, giving you the best chance of winning the point. 
 
 ### Why you?
-_What excites you about this idea? How did you come up with it?_
-
+_What excites you about this DSL? How did you come up with it?_
+I grew up playing competitive badminton at the provincial and national level (some of my friends continue to play professionally) so when a classmate mentioned the act of hailing a taxi as a DSL I immediately thought of this. The thrill of intense, long rallies in badminton is incomparable to anything else. It gives you this incredibly giddy feeling. 
 
 ### Domain
 _Describe the project's domain in five words._
-
+Communicating secretly in badminton doubles
 
 ### Interface (syntax)
-_How might the user interact with the language? What does programming look 
+_How might the user interact with the language? What does using the DSL look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+We used to have multiple different signals: 1 for a regular serve, 2 for a high serve, 3 for a serve to the side etc. Serving to the side would catch the opponent off guard and limits the angles that they can return (although it is slightly slower so it cannot be constantly used). If done right, your partner can also shift over to put even more pressure on the opponent to hit an inch-perfect shot (forcing errors). Even if we were doing a normal serve, it's important to continue the habit of giving the hand sign to maintain the deception. 
+
+There's no wrong way to communicate in badminton, just that if you let the opponent know what you're going to do beforehand, you're not going to win the point. So this is definitely the best method for serving. 
 
 ### Operation (semantics)
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
 
+N/A
```

Comment:
Perhaps, when a program runs, your partner notices the hand signal, and in response to the input, they move to a certain part of the field to have a better reaction time, as you mentioned.